### PR TITLE
admin feature flags

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [FE PR #922](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/922)

If not seeing Administrator Dashboard section, for Admin, after migrate:
add `user_id: 2` `group_id: 31` in table `auth_user_groups` to add `superuser` to admin user. 

~Don't currently have a seed file set up, so add the following data to a row in `feature_flags_featureflags`table after `docker-compose up`.~ "seeding" from FE config file

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
🤞 Ready for review! 🤞

File: talentmap_api/feature_flags/views/featureflags.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import datetime
+from django.shortcuts import get_object_or_404
+# from django.http import Http404
```
Comment (mjoyce91) [inline talentmap_api/feature_flags/views/featureflags.py:3]: needed?

File: talentmap_api/feature_flags/views/featureflags.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+class FeatureFlagsView(mixins.RetrieveModelMixin,
+                       mixins.UpdateModelMixin,
+                       mixins.CreateModelMixin,
+                       GenericViewSet,
+                       APIView):
+
+    serializer_class = FeatureFlagsSerializer
+    permission_classes = (IsAuthenticatedOrReadOnly, isDjangoGroupMemberOrReadOnly('superuser'))
+    # permission_classes = (IsAuthenticatedOrReadOnly, isDjangoGroupMemberOrReadOnly('sdkfsjkhf'))
```
Comment (mjoyce91) [inline talentmap_api/feature_flags/views/featureflags.py:31]: needed?

File: talentmap_api/feature_flags/views/featureflags.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+from talentmap_api.common.common_helpers import in_group_or_403
+
+
+from talentmap_api.feature_flags.models import FeatureFlags
+from talentmap_api.feature_flags.serializers.featureflags import FeatureFlagsSerializer
+
+import logging
+logger = logging.getLogger(__name__)
+
+from pprint import pprint
```
Comment (mjoyce91) [inline talentmap_api/feature_flags/views/featureflags.py:21]: ❌

File: talentmap_api/feature_flags/views/featureflags.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            return Response(queryset[0].feature_flags)
+
+
+    def perform_create(self, request):
+        in_group_or_403(self.request.user, f"superuser")
+        # in_group_or_403(self.request.user, f"sdfsdf")
+        instance = FeatureFlags()
+        logger.info(f"User {self.request.user.id}:{self.request.user} creating feature_flags entry {instance}")
+        instance.feature_flags = request.data
+        pprint(instance.feature_flags)
```
Comment (mjoyce91) [inline talentmap_api/feature_flags/views/featureflags.py:53]: ❌

File: talentmap_api/feature_flags/views/featureflags.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        queryset = FeatureFlags.objects.order_by('-date_updated')
+        if not queryset:
+            return Response(status=status.HTTP_404_NOT_FOUND)
+        else:
+            return Response(queryset[0].feature_flags)
+
+
+    def perform_create(self, request):
+        in_group_or_403(self.request.user, f"superuser")
+        # in_group_or_403(self.request.user, f"sdfsdf")
```
Comment (mjoyce91) [inline talentmap_api/feature_flags/views/featureflags.py:49]: pprint

File: talentmap_api/feature_flags/views/featureflags.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        instance = FeatureFlags()
+        logger.info(f"User {self.request.user.id}:{self.request.user} creating feature_flags entry {instance}")
```
Comment (mjoyce91) [inline talentmap_api/feature_flags/views/featureflags.py:51]: Let's keep this, this is good info to have in the logs

File: talentmap_api/feature_flags/views/featureflags.py [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+    def retrieve(self, request, pk=None, format=None):
+        '''
+        Gets the Feature Flags file
+        '''
+        queryset = FeatureFlags.objects.order_by('-date_updated')
+        if not queryset:
+            return Response(status=status.HTTP_404_NOT_FOUND)
+        else:
+            return Response(queryset[0].feature_flags)
```
Comment (mjoyce91) [inline talentmap_api/feature_flags/views/featureflags.py:44]: 👍

File: talentmap_api/feature_flags/views/featureflags.py (hunk 4 above)
Comment (elizabeth-jimenez) [reply talentmap_api/feature_flags/views/featureflags.py:53]: 😭

File: talentmap_api/feature_flags/views/featureflags.py (hunk 1 above)
Comment (elizabeth-jimenez) [reply talentmap_api/feature_flags/views/featureflags.py:3]: no :(

File: talentmap_api/feature_flags/views/featureflags.py (hunk 5 above)
Comment (mjoyce91) [reply talentmap_api/feature_flags/views/featureflags.py:49]: >pprint

Idk why I commented that - must have copy pasted. In either case I think we can remove this comment

**Pull request merged**